CHANGES.txt

```diff
@@ -1,4 +1,51 @@
-﻿NUnit 3.4.1 - June 30, 2016
+﻿NUnit 3.5 - October 3, 2015
+
+This is the first version of NUnit where the framework will be released separately from the
+console runner, engine and other extensions. From this point forward, the NUnit Framework will be
+released on its own schedule that is not bound to that of any other NUnit project and version numbers
+may diverge over time.
+
+This is also the first release where the NUnit Framework will not be included in the installer. Only
+the console runner, engine and extensions will be available as an MSI installer. We recommend that you
+use the NUnit NuGet packages for the framework, but a ZIP file with the binaries will also be available.
+
+Framework
+
+ * Added Assert.Zero and Assert.NotZero methods
+ * Tests with a Timeout will no longer timeout while you are debugging
```

Comment:
I usually put any feature changes likely to require documentation updates here and then use it to drive the docs updates. By that criterion, I think #144 and #1553 qualify.